TypeCobol/Compiler/CompilationProject.cs

```diff
@@ -79,12 +76,21 @@ public static CompilationProject Load(string projectName, string rootDirectory)
         ICobolLibrary rootDirectoryLibrary;
 
         // Default properties for all files of the project
-        public Encoding Encoding { get; private set; }
-        public EndOfLineDelimiter EndOfLineDelimiter { get; private set; }
-        public int FixedLineLength { get; private set; }
-        public ColumnsLayout ColumnsLayout { get; private set; }
-        public TypeCobolOptions CompilationOptions { get; private set; }
-        public IAnalyzerProvider AnalyzerProvider { get; private set; }
+        public Encoding Encoding => Format.Encoding;
+        public EndOfLineDelimiter EndOfLineDelimiter => Format.EndOfLineDelimiter;
+        public int FixedLineLength => Format.FixedLineLength;
+        public ColumnsLayout ColumnsLayout => Format.ColumnsLayout;
```

Comment:
Why keep that ? It looks strange from the user point-of-view because nothing indicates that `CompilationProject.Encoding` is the same as `CompilationProject.Format.Encoding` for example. Please remove these 4 properties, there are only a few usages.

Reply:
Removed in [8d9c2da](https://github.com/TypeCobolTeam/TypeCobol/pull/2158/commits/8d9c2daba074a18dbf5260d351d3278d470ebf44)